src/store/user/user.zod.ts

```diff
@@ -46,9 +47,13 @@ export const ManageUser = z.object({
   last_login: z.string().nullable(),
   photo_count: z.number(),
   save_metadata_to_disk: z.string(),
-  scan_directory: z.string(),
+  scan_directory: z.string().optional().nullable(),
```

Comment:
is it possible to use zod's [nullish](https://github.com/colinhacks/zod#nullish) here?